src/services/ngx-uploader.ts

```diff
@@ -30,13 +30,18 @@ export class NgUploaderService {
 
   uploadFile(file: any): void {
     let xhr = new XMLHttpRequest();
-    let form = new FormData();
+    let payload: any;
```

Comment:
Had to make an additional variable, as `FormData | File` doesn't support `.append` method. Can you review it now?